presto-thrift-interface/src/main/java/com/facebook/presto/thrift/interfaces/readers/ReaderUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.thrift.interfaces.readers;
+
+import com.facebook.presto.thrift.interfaces.client.ThriftColumnData;
+
+import java.util.List;
+
+public final class ReaderUtils
+{
+    private ReaderUtils()
+    {
+    }
+
+    public static ThriftColumnData columnByName(List<ThriftColumnData> columnsData, String columnName)
```

Comment:
Both the current and new signature look kind of the same to me. Any advantage of changing it?